rinja/Cargo.toml

```diff
@@ -32,7 +32,7 @@ with-rocket = ["rinja_derive/with-rocket"]
 with-warp = ["rinja_derive/with-warp"]
 
 [dependencies]
-rinja_derive = { version = "0.2.0", path = "../rinja_derive" }
+rinja_derive = { version = "=0.3.0", path = "../rinja_derive" }
```

Comment:
Why the equal sign?

Reply:
The problem is, should we make changes to the parser (say, adding `c"…"` string support), which require changes to the exported enums, then we need to do a semver incompatible release. If we want to keep the version numbers of all three crates the same, then we would need to bumb rinja's version number, too, making things difficult for downstream users.

With the equal signs we can control that all three versions stay related, and that a user cannot do a partial update.

Reply:
Fine by me!